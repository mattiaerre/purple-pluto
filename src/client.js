import tracker from './tracker';
import painter from './painter';

const client = (options) => { // eslint-disable-line arrow-body-style
  return {
    tracker: tracker(options.tracker),
    painter: painter(options.painter)
  };
};

export default client;
