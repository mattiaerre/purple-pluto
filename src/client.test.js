import client from './client';

test('client', () => {
  const options = { tracker: { target: {} }, painter: {} };
  expect(client(options)).toMatchSnapshot();
});
