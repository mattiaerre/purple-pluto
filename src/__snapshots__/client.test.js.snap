// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`client 1`] = `
Object {
  "painter": Object {
    "paint": [Function],
  },
  "tracker": Object {
    "getData": [Function],
    "reset": [Function],
    "start": [Function],
    "stop": [Function],
  },
}
`;
